org.eevolution.warehouse/src/main/java/base/org/spin/form/OutBoundOrder.java

```diff
@@ -1152,14 +1381,14 @@ public void loadBuffer(IMiniTable orderLineTable) {
 		int rows = orderLineTable.getRowCount();
 		int m_C_OrderLine_ID = 0;
 		BigDecimal qty = Env.ZERO;
-		Integer seqNo = 0;
+		BigDecimal seqNo = BigDecimal.ZERO;
```

Comment:
Please make this code more atomic, the refactory can be change in other pull request as Source code Improve